pmm-app/src/pmm-check/CheckPanel.tsx

```diff
@@ -46,12 +46,12 @@ export const CheckPanel: FC = () => {
     {
       label: Messages.failedChecksTitle,
       key: TabKeys.failedChecks,
-      component: <FailedChecksTab hasNoAccess={hasNoAccess} />,
+      component: <FailedChecksTab key="failed-checks" hasNoAccess={hasNoAccess} />,
     },
     {
       label: Messages.allChecksTitle,
       key: TabKeys.allChecks,
-      component: <AllChecksTab />,
+      component: <AllChecksTab key="all-checks" />,
```

Comment:
Why do we need `key` here? Was it `data-qa` instead?

Reply:
(same for `FailedChecksTab `)

Reply:
because those are rendered conditionally in a `map`, React complains if there's no `key`